src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToolTipTests/ToolTip_Tests.cs

```diff
@@ -13,14 +13,14 @@ public partial class ToolTip_Tests : SampleControlUITestBase
 	{
 		[Test]
 		[AutoRetry]
-		[ActivePlatforms(Platform.iOS, Platform.Browser)] // Android is disabled https://github.com/unoplatform/uno/issues/1630
+		[ActivePlatforms(Platform.iOS)] // Android is disabled https://github.com/unoplatform/uno/issues/1630
```

Comment:
We should not be removing a working test. If we need to make an iOS specific set of calls, let's change those for the platform.  You can use `AppInitializer.GetLocalPlatform()` to determine if the test is running for iOS and invoke the appropriate test method (FastTap or TouchAndHold).

Reply:
There isn't appropriate function to show tooltip in Browser environment. The reason that `TouchAndHold()` works on iOS, but not on Browser is because Browser closes tooltip when calling `TouchAndHold()` .
https://github.com/unoplatform/uno/pull/12419/commits/390aee5e33c9e3fcf48748079f60c82aa860fbe4#diff-ef1649477ba6ac8e50556565d5ce6fba198a7c4630d0e13a9f6e8d76ec26d4f1R274

Reply:
Further, TouchAndHold functions have been not implemented yet.
https://github.com/unoplatform/Uno.UITest/blob/64c6caccd4b2ede96a6ada265c795acb640b3411/src/Uno.UITest.Puppeteer/SeleniumApp.cs#L500-L502

Reply:
@jeromelaban , What should I do in this case?

Reply:
I see. On the browser you can use `DragCoordinates`, from coordinates outside the button to the inside of the button. It should have the effect of moving the pointer without clicking down.